src/Files.App/Actions/Show/ToggleSidebarAction.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) 2024 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+namespace Files.App.Actions
+{
+	internal sealed class ToggleSidebarAction : ObservableObject, IToggleAction
+	{
+		private readonly IAppearanceSettingsService AppearanceSettingsService = Ioc.Default.GetRequiredService<IAppearanceSettingsService>();
+
+		public string Label
+			=> "ToggleSidebar".GetLocalizedResource();
+
+		public string Description
+			=> "ToggleSidebarDescription".GetLocalizedResource();
+
+		public HotKey HotKey
+			=> new(Keys.S, KeyModifiers.CtrlAlt);
```

Comment:
Is this shortcut based on anything in particular?

Reply:
The keyboard modifier for toggling the sidebar in VS Code is <kbd>Ctrl+B</kbd>, I think that should apply here.

Reply:
I think the original action in Files used to be `Ctrl+Shift+B`, but that's since been taken by the action to hide the toolbar. Let's go with `Ctrl+B`.

Reply:
> Is this shortcut based on anything in particular?

[Quite](<https://github.com/files-community/Files/commit/720e3b247fbe1c126c4209157301b8d6e27b1a3c>).

Furthermore, it seems to be in agreement with the currently implemented style of shortcuts for showing other panes in [/Files/src/Files.app/Actions/Show](<https://github.com/files-community/Files/tree/main/src/Files.App/Actions/Show>): `[Ctrl]+[Alt]+[%K%]`, where `%K%` shares the capital letter with the pane's title.

Reply:
Currently they're all Control, Alt and:
- D: Details
- I: Info
- P: Preview
- H: Hidden
- B: ToolBar

Reply:
> Anybody who cares will rebind to what they personally prefer anyway.

That's true, but wherever possible, we aim to use known shortcuts to improve discoverability.

Reply:
And that's the shortcut known from a previous implementation.  
I'll say that if this predefined shortcut needs to be changed — it's better done separately as a revamp of the whole schema it's following.

Reply:
It's been a while since we've had this action and I think that at this point in time, the shortcut in Visual Studio Code is better known than what we had in the past.

Reply:
Starting with just this one, leaving all the rest as `[Ctrl]+[Alt]+[%capital%]` for later?